internal/featuregates/featuregates.go

```diff
@@ -13,12 +15,17 @@ type FeatureGates struct {
 	ConfigMapName string
 }
 
+// FeatureGate Status Struct map of string and bool
+type FeatureGateStatus map[string]bool
+
 var DefaultFeatureGates = map[string]bool{
```

Comment:
I think it would make sense if `DefaultFeatureGates` was of the `FeatureGateStatus` type, defined just above, wouldn't it?

Reply:
Right.